### DISCUSSION TITLE
How to use "Allowed Internet Addresses"？Can we get some image

### DISCUSSION THREAD

**Discussion by lonesoul003** [tnodir/fort#235] [Q&A]
I know how to use "internet address" 
if I set include 192.168.1.1
then I cannot ping it in CMD
chrome can't open https://192.168.1.1  either.

 but I confused it very much 
How to use "Allowed Internet Addresses"
 Both cmd and chrome are not affected

**Comment by tnodir** [comment]
Please first read the following:
- [Blocking IP addresses or ranges of IP addresses](https://github.com/tnodir/fort/wiki/User-Guide#blocking-ip-addresses-or-ranges-of-ip-addresses)
- [What is the difference between "Internet Addresses" and "Allowed Internet Addresses" on the "IP Addresses" tab?](https://github.com/tnodir/fort/wiki/FAQ#what-is-the-difference-between-internet-addresses-and-allowed-internet-addresses-on-the-ip-addresses-tab)
- [Do the App rules override the Allowed Internet rules?](https://github.com/tnodir/fort/wiki/FAQ#do-the-app-rules-override-the-allowed-internet-rules)
